openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateFanVariableVolume.cpp

```diff
@@ -181,10 +181,12 @@ boost::optional<IdfObject> ForwardTranslator::translateFanVariableVolume( FanVar
     }
   }
 
+  // End Use Subcategory
+  idfObject.setString(Fan_VariableVolumeFields::EndUseSubcategory, modelObject.endUseSubcategory());
```

Comment:
@jmarrec should you check that this is set first?  what is the default value?

Reply:
The default value is "General". You merged, but I can use isEndUseSubcategoryDefaulted if you want @macumber

Reply:
Nah, I remembered we decided to put os defaults in the idf in case they diverge from e+